src/main/java/com/knighttodo/knighttodo/domain/TodoVO.java

```diff
@@ -13,7 +13,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-@ToString(exclude = {"routine", "block"})
```

Comment:
Why did you remove it? The idea was that we want to display entity + its children to avoid cycling in toString

Reply:
moved it back.